.travis.yml

```
@@ -1,32 +1,44 @@
+---
 language: node_js
-env:
-  global:
-    - CC_TEST_REPORTER_ID=ee7700d6670416f453c04be8c8ea32b756b7309ce33bf965402136f5d4fb91bc
-    - SAUCE_USERNAME=nomego
-    - secure: BxvbM9LnQsXLoaohrIf+ip1kSVmqTgqOxwGU7HZnaw3hOjOZ5PGaD2l1G9X0j2c4QysB6KLcbmrJDWkIVtpiIR2NgC9DQbDpgCyafO+DXk6+sVOuGuV1h/5yt+rfx+VZOItkfCcmsi0MgUjCNCpk22lHIwDbRY3+vvRNNgF8eyyCMz47N+TBzpinTohU0xZOpWHPQbFQ6TfR6heoV/DrF/XPkDbDDLU4WV9AWrPZk9dHzLZLbl5DyXwWivyNqSCkPQCBhYckqZRuMU0XWNlp2tuxC+UMjBDh9HFWl4swZfcrBPOUE0zhjxKBF88QzJyfsLNd5pvkJNKYeQsLZ9z0I47INX867ZGi2iIAX4JW1I4FbAwZs8YnCMAKUzOdJgF8WkvjWRFZ+teO8Wp8s+ymb5NZYMaXlbVjg2wVYau3+1agLhhNXwVv7vYAfLclLfXHKrRCPeSBkX7iVUwlOn5OemWNJBJ9eGCqYi/mbU900wIdk00g9nyZeEjwtbWZkjrMprWqQkCPWxA5ZmkLWlm1pkduokUx0LlCV5MigOgJkQb//OacYwLjIdJCKX/V/KOipCCAlWBaVh3cFBQAx3EcPelD1nWT07xQtcz/BwrmvMGaNnQln3RvuOAV3oRfltR1EDIcRStoDNz00Evc5+Q1DL4fkHnMYWYFsfIQorlb+6o=
+node_js: "lts/*"
+
 addons:
   firefox: latest
   apt:
     sources:
       - google-chrome
     packages:
-      - google-chrome-stable
+      - google-chrome-unstable
+
 cache:
   yarn: true
   directories:
     - node_modules
     - bower_components
-    - ".eslintcache"
-    - "$HOME/.cache/bower"
+    - .eslintcache
+    - $HOME/.cache/bower
+    - $HOME/cc
+
+env:
+  global:
+    - CC_TEST_REPORTER_ID=ee7700d6670416f453c04be8c8ea32b756b7309ce33bf965402136f5d4fb91bc
+    - SAUCE_USERNAME=nomego
+    - secure: BxvbM9LnQsXLoaohrIf+ip1kSVmqTgqOxwGU7HZnaw3hOjOZ5PGaD2l1G9X0j2c4QysB6KLcbmrJDWkIVtpiIR2NgC9DQbDpgCyafO+DXk6+sVOuGuV1h/5yt+rfx+VZOItkfCcmsi0MgUjCNCpk22lHIwDbRY3+vvRNNgF8eyyCMz47N+TBzpinTohU0xZOpWHPQbFQ6TfR6heoV/DrF/XPkDbDDLU4WV9AWrPZk9dHzLZLbl5DyXwWivyNqSCkPQCBhYckqZRuMU0XWNlp2tuxC+UMjBDh9HFWl4swZfcrBPOUE0zhjxKBF88QzJyfsLNd5pvkJNKYeQsLZ9z0I47INX867ZGi2iIAX4JW1I4FbAwZs8YnCMAKUzOdJgF8WkvjWRFZ+teO8Wp8s+ymb5NZYMaXlbVjg2wVYau3+1agLhhNXwVv7vYAfLclLfXHKrRCPeSBkX7iVUwlOn5OemWNJBJ9eGCqYi/mbU900wIdk00g9nyZeEjwtbWZkjrMprWqQkCPWxA5ZmkLWlm1pkduokUx0LlCV5MigOgJkQb//OacYwLjIdJCKX/V/KOipCCAlWBaVh3cFBQAx3EcPelD1nWT07xQtcz/BwrmvMGaNnQln3RvuOAV3oRfltR1EDIcRStoDNz00Evc5+Q1DL4fkHnMYWYFsfIQorlb+6o=
+    - NEO_CC_URL=https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
+    - NEO_CC=$HOME/cc/cc-test-reporter
+    - SELENIUM_OVERRIDES_CONFIG=$TRAVIS_BUILD_DIR/selenium-overrides.js
+    - LAUNCHPAD_CHROME=/usr/bin/google-chrome-unstable
+    - TZ=Europe/Stockholm
+
 before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
-  - yarn run lint
+  - yarn lint
+  - ([[ -e $NEO_CC ]] || curl -L $NEO_CC_URL > $NEO_CC) && chmod +x $NEO_CC
+  - $NEO_CC before-build
+
 script:
-  - yarn test
-after_script:
-  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+  - yarn test $([ -z "$SAUCE_ACCESS_KEY"] && echo "--skip-plugin sauce")
```

```suggestion
  - yarn test $([ -z "$SAUCE_ACCESS_KEY" ] && echo "--skip-plugin sauce")
```

Otherwise this causes an error in the build.. should be updated everywhere

---

As seen here: https://travis-ci.com/Neovici/cosmoz-tabs/jobs/197139338#L522

` /home/travis/.travis/functions: line 104: [: missing `]'`